go.mod

```diff
@@ -1,7 +1,6 @@
 module github.com/kubernetes-sigs/ingate
 
-go 1.24
-
+go 1.24.1
```

Comment:
I believe we want to keep the patch version, and let it be the latest patch in the build process.